composer.json

```
@@ -13,12 +13,15 @@
     ],
     "require": {
         "php": ">=5.6.0",
-        "symfony/symfony": "~2.3|~3.0",
+        "symfony/config": "~2.3 || ~3.0 || ^4.0",
+        "symfony/dependency-injection": "~2.3 || ~3.0 || ^4.0",
+        "symfony/http-kernel": "~2.3 || ~3.0 || ^4.0",
         "novaway/open-graph": "^1.0"
     },
     "require-dev": {
-        "atoum/atoum": "^2.5|^3.0",
-        "atoum/stubs": "^2.5"
+        "atoum/atoum": "^2.5 || ^3.0",
```

The Travis error seems to be related to the oldest versions of atoum, not compatible with recent PHP versions.
Can you try this fix : 
```suggestion
        "atoum/atoum": "^3.0",
```

By the way `"symfony/symfony": "~2.3 || ~3.0 || ^4.0"` is not useful in `require-dev` because it's already in the `require` section.

---

Well, I changed the requirements to single symfony package in the `require` section.
Unfortunately the test failed only with `symfony/framework-bundle` in `require-dev` so I required `symfony/symfony` there.